src/databricks/labs/ucx/assessment/queries/azure_service_principals.sql

```diff
@@ -0,0 +1,10 @@
+-- viz type=table, name=Azure SPNs, columns=application_id,secret_scope,secret_key,tenant_id,storage_account
+-- widget title=Azure SPNs, col=0, row=41, size_x=6, size_y=8
+select
```

Comment:
attach a screenshot of the dashboard with this widget.

Reply:
<img width="865" alt="image" src="https://github.com/databrickslabs/ucx/assets/106815134/34fb03bd-7274-4b89-8716-1615b826a7c6">

Reply:
<img width="1721" alt="image" src="https://github.com/databrickslabs/ucx/assets/106815134/7fc46cc1-0c7e-46b5-894b-eaa564871d79">
<img width="867" alt="image" src="https://github.com/databrickslabs/ucx/assets/106815134/f978ec37-0113-4699-90a3-863ef038ccf9">
<img width="1719" alt="image" src="https://github.com/databrickslabs/ucx/assets/106815134/a04ca710-7a76-42b6-bfa1-a642eec741fd">
<img width="1713" alt="image" src="https://github.com/databrickslabs/ucx/assets/106815134/bd2480b8-51a1-468d-bcee-a0ca12710a58">